autoload/dbext_dbi.vim

```diff
@@ -1742,6 +1943,239 @@ sub db_results_list
     return 0;
 }
 
+
+sub db_enable_srv_out
+{
+
+  # see ~/Documents/0.mop/Perl_Oracle_DBD_dbms_output/dbms_output_test_case_1.vim
+  # for automated test case:
+  # vim -c 'source ~/Documents/0.mop/Perl_Oracle_DBD_dbms_output/dbms_output_test_case_1.vim|call Test_Case_1()'
+
+  my ($conn_local, $driver, $z_user_data) = db_get_connection();
+  if ($z_user_data->{z_srv_out_enabled}) {
+    db_debug("skip <enable> of already enabled server output");
+  }
+  else {
+    $conn_local->func( 1000000, 'dbms_output_enable' );
```

Comment:
Would it make sense for this to be a variable so that it can be customized easily?

Reply:
That would make perfectly sense but I have to find time yet to work on it. Next 2-3 weeks are quite tough on my side so I push now to get at least quick changes you requested done.